DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Dashboard/Index.cshtml

```diff
@@ -0,0 +1,230 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Dashboard
+@using Microsoft.Extensions.Configuration
+@model CentreDashboardViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/centreDashboard.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Centre dashboard";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/tracking/dashboard";

```

Comment:
There have been a number of changes to use AppRootPath rather than CurrentSystemBaseUrl in the HeaderPath. From what I recall, AppRootPath is for this new site, so if these are supposed to link to this page, I think this line (and the other similar changes) should be
ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard